app.rb

```diff
@@ -0,0 +1,102 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'person'
+
+class App
+  attr_accessor :books, :student, :teacher, :rental, :person
+
+  def initialize
+    @books = []
+    @student = []
+    @teacher = []
+    @rental = []
+  end
+
+  # list all books
+  def list_books
+    @books.each_with_index { |_books, _i| print " Title: \"#{book.title}\", Author: \"#{book.author}\"\n" }
```

Comment:
- [ ] As I have said earlier, I am getting this error when I tried to get the list of all books because of the spelling error👇🏽 
![image](https://user-images.githubusercontent.com/63473584/194676436-1badd013-28fd-4132-89f0-de3507754dc4.png)

You should have `book` instead of ` _books_` in the each loop

e.g
` @books.each_with_index { |book, i| print " Title: \"#{book.title}\", Author: \"#{book.author}\"\n" }`

Reply:
Hi!
Thanks for your review, however, the rubocop linters keeps changing the code. Is there any other thing I can do because I  have only one more attempt to submit